test/outputers/html_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule WhiteBread.Outputers.HTMLTests do
+	use ExUnit.Case
+  alias WhiteBread.Outputers.HTML
+  alias WhiteBread.Gherkin.Elements.Scenario
+
+  @console WhiteBread.Outputers.Console
+
+  test "default outputer is for the console" do
+     assert @console = Application.fetch_env! :white_bread, :outputer
+  end
+
+  test "file path fetched on initialization" do
+    old = Application.fetch_env! :white_bread, :path
+    :ok = Application.put_env :white_bread, :path, "/fu/bar.baz"
```

Comment:
Keeping track of/restoring an app env variable looks like something to be put into `setup`